aiographite/aiographitesend.py

```diff
@@ -0,0 +1,449 @@
+import asyncio
+from aiographite.graphite_escaping import metrics_name_to_graphite, metrics_name_from_graphite
+import pickle
+import struct
+import socket
+import os
+import time
+
+
+DEFAULT_GRAPHITE_PLAINTEXT_PORT = 2003
+DEFAULT_GRAPHITE_PICKLE_PORT = 2004
+SUPPORT_PROTOCOLS = ["plaintext", "pickle"]
+
+
+class AioGraphiteSendException(Exception):
+    pass
+
+
+class AsyncioGraphiteSendService(object):
+
+	def __init__(self, graphite_server, graphite_port, protocol = "pickle"):
+
+		self.graphite_server_address = (graphite_server, graphite_port)
```

Comment:
private / protected variables should have a leading undescore:

```
self._graphite_server_address
```

this goes for both fields and methods:
```
self._connect_to_graphite()
```
It helps future developers understand what they can and can't modify. If everything is "public" developers don't really know what they could refactor without breaking others, and consumers assume they can use everything.